CIP-0025/README.md

```diff
@@ -78,7 +78,20 @@ The structure allows for multiple token mints, also with different policies, in
zRhNS45Miw1LjkyLDAsMSwwLDcuOTUsMi42MkE1LjkxLDUuOTEsMCwwLDAsMTk4LjQ5LDIzMi43NFoiIGZpbGw9IiMwMDMzYWQiLz48cGF0aCBkPSJNMTI5LjczLDE3M2ExOC4zLDE4LjMsMCwxLDAtMTUuMzIsOC4yN0ExOC4yMSwxOC4yMSwwLDAsMCwxMjkuNzMsMTczWiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik05My4xNiwyMDEuMDlhOS4xNCw5LjE0LDAsMSwwLDIuNjQsMTIuNjZBOS4xNSw5LjE1LDAsMCwwLDkzLjE2LDIwMS4wOVoiIGZpbGw9IiMwMDMzYWQiLz48cGF0aCBkPSJNMTM1LjgzLDIzNi4xMmE3LjUzLDcuNTMsMCwxLDAsNy4wOSw3Ljk1QTcuNTMsNy41MywwLDAsMCwxMzUuODMsMjM2LjEyWiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik0xMzUuOTEsMTg4Ljc0QTEwLjc2LDEwLjc2LDAsMSwwLDE0NiwyMDAuMDksMTAuNzUsMTAuNzUsMCwwLDAsMTM1LjkxLDE4OC43NFoiIGZpbGw9IiMwMDMzYWQiLz48cGF0aCBkPSJNMTc5LjkyLDIwMC4zNmE5LjE1LDkuMTUsMCwxLDAsMTIuMjksNEE5LjEzLDkuMTMsMCwwLDAsMTc5LjkyLDIwMC4zNloiIGZpbGw9IiMwMDMzYWQiLz48cGF0aCBkPSJNMzgxLjg2LDgyLjM1YzE3LjcyLDAsMzAuMTMsOS43NCwzMC4xMywyNy44Mkg0MzUuMmMwLTMxLjcyLTIzLjM5LTQ3LTUzLjY5LTQ3LTM3LjIyLDAtNTguODQsMjAuMjEtNTguODQsNjIuOTIsMCw0My43NywyMS42Miw2Mi45MSw1OC44NCw2Mi45MSwzMC42NiwwLDUzLjY5LTE0Ljg4LDUzLjY5LTQ2LjYxSDQxMi44N2MwLDE3LjcyLTEyLjIzLDI3LjQ3LTMxLDI3LjQ3LTI0LjY0LDAtMzYtMTMuNjUtMzYtNDJ2LTMuNTRDMzQ1Ljg5LDk2LjUyLDM1Ny40LDgyLjM1LDM4MS44Niw4Mi4zNVoiIGZpbGw9IiMwMDMzYWQiLz48cGF0aCBkPSJNNDk2LjU3LDY1LjMzLDQ0OS4yNSwxODYuOTFoMjMuNGw5Ljc0LTI2LjQxSDUzOEw1NDgsMTg2LjkxaDI0LjQ2TDUyNS4xMSw2NS4zM1ptLTcuMDksNzZMNTAxLjg5LDEwOGMyLjQ4LTYuNTUsNi45MS0yMC43Myw3LjgtMjMuMjFoLjg4Yy44OSwyLjQ4LDUuMzIsMTYuODMsNy44LDIzLjIxbDEyLjQsMzMuMzJaIiBmaWxsPSIjMDAzM2FkIi8+PHBhdGggZD0iTTY5OSwxMDIuMTljMC0yMi41LTEyLjk0LTM2Ljg2LTM5LjctMzYuODZINTk1LjE0VjE4Ni45MWgyMi42OXYtNDdoMzcuNzRsMjMsNDdoMjQuODFsLTI2LTUwLjUxQzY5MS43MywxMzAuNzMsNjk5LDExOC42OCw2OTksMTAyLjE5Wk02NTcuNTIsMTIwLjhINjE3LjgzVjg0LjY1aDM5LjY5YzEyLjIzLDAsMTguNjEsNi41NiwxOC42MSwxNy41NEM2NzYuMTMsMTEzLjcyLDY2OS4yMiwxMjAuOCw2NTcuNTIsMTIwLjhaIiBmaWxsPSIjMDAzM2FkIi8+PHBhdGggZD0iTTc3OC41OCw2NS4zM0g3MzBWMTg2LjkxaDQ4LjU2YzM2LjY4LDAsNTktMTkuMTQsNTktNjAuNzlTODE1LjI2LDY1LjMzLDc3OC41OCw2NS4zM1ptMzUuOCw2Mi41NmMtLjE4LDI2LjU5LTEyLjc2LDM5LjctMzUuOCwzOS43SDc1Mi43MVY4NC42NWgyNS44N2MyMywwLDM1LjgsMTMuMTIsMzUuOCwzOS43WiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik04OTcsNjUuMzMsODQ5LjY2LDE4Ni45MWgyMy4zOWw5Ljc1LTI2LjQxaDU1LjY1bDkuOTIsMjYuNDFoMjQuNDZMOTI1LjUxLDY1LjMzWm0tNy4wOSw3Nkw5MDIuMywxMDhjMi40OC02LjU1LDYuOTEtMjAuNzMsNy44LTIzLjIxSDkxMWMuODgsMi40OCw1LjMyLDE2LjgzLDcuOCwyMy4yMWwxMi40LDMzLjMyWiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik0xMDc2LjU0LDEzNy42NGMwLDIuMy4zNSw4LjUuMzUsOS41N2wtLjg4LjM1Yy0uNTMtLjctNC4wOC02LjkxLTYtOS43NGwtNTIuODEtNzIuNDlIOTk1LjU1VjE4Ni45MWgyMS44VjExNC4wN2MwLTMtLjM2LTguODYtLjM2LTkuNzVsLjg5LS41M2MuNTMuNzEsMy4xOSw1Ljg1LDYsOS45M2w1My4xNyw3My4xOWgyMS4yNlY2NS4zM2gtMjEuNzl2NzIuMzFaIiBmaWxsPSIjMDAzM2FkIi8+PHBhdGggZD0iTTExODguNjgsNjMuMjFjLTM4LjEsMC02MS4xNCwyMC4yLTYxLjE0LDYyLjkxczIzLDYyLjkxLDYxLjE0LDYyLjkxLDYxLjMyLTIwLjIsNjEuMzItNjIuOTFTMTIyNi43OCw2My4yMSwxMTg4LjY4LDYzLjIxWm0zOC4xLDY0Ljg2YzAsMjcuODItMTMuMjksNDEuODItMzguMSw0MS44Mi0yNC42MywwLTM3LjkyLTE0LTM3LjkyLTQxLjgydi0zLjcyYzAtMjcuODMsMTMuNDctNDIsMzcuOTItNDIsMjQuNjQsMCwzOC4xLDE0LjE3LDM4LjEsNDJaIiBmaWxsPSIjMDAzM2FkIi8+PC9nPjwvZz48L3N2Zz4=`
```

Comment:
```suggestion
```
The section of included code doesn't say anything additional about the property usage beyond repeating what you already quoted (`data:<mime_type>;base64`).  It takes a disproportionate part of the screen & the document itself, especially on small screen devices (we do have CIP readers on those, as we found out with another CIP).

What about another link, more user friendly than the RFC, that goes into more detail about broader usage of image types, compatibility, and constructing the base64 data?  https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs